CONFIGURATION.md

```diff
@@ -25,6 +25,23 @@ modules:
 
 ```
 
+### Address selection
+
+Only a single connection attempt is made to the target under test - there is
+never any "happy eyeballs" style of connection retry.  If the target is
+specified using a hostname, and that hostname resolves to multiple
+addresses, then a single address is selected to test according to the
```

Comment:
This is also true if an host lookup returns multiple A or AAAA records. We've had similar support issues in the past where users expect different behavior when multiple A records are returned.

```suggestion
addresses or IP protocols, then a single address is selected to test according to the
```